doc/Advanced.md

```diff
@@ -992,8 +992,77 @@ The same approach can be applied to other actors in the QC framework, like Check
 
 ## Definition and access of task-specific configuration
 
+A task can access custom parameters declared in the configuration file at `qc.tasks.<task_id>.extendTaskParameters` or `qc.tasks.<task_id>.taskParameters`. They are stored inside an object of type `CustomParameters` named `mCustomParameters`, which is a protected member of `TaskInterface`.
+
+The simple, deprecated, syntax is 
+```json
+    "tasks": {
+      "QcTask": {
+        "taskParameters": {
+          "myOwnKey": "myOwnValue"
+        },
+```
+It is accessed with : `mCustomParameters["myOwnKey"]`. 
+
+The new syntax is
+```json
+    "tasks": {
+      "QcTask": {
+        "extendedTaskParameters": {
+          "default": {
+            "default": {
+              "myOwnKey": "myOwnValue",
+              "myOwnKey2": "myOwnValue2",
+              "myOwnKey3": "myOwnValue3"
+            }
+          },
+          "physics": {
+            "default": {
+              "myOwnKey1": "myOwnValue1b",
+              "myOwnKey2": "myOwnValue2b"
+            },
+            "pp": {
+              "myOwnKey1": "myOwnValue1c"
+            },
+            "PbPb": {
```

Comment:
Are "pp" and "PbPb" the correct examples of the strings to be used? The same comment for "physics" and "cosmics".

Reply:
I have added an example for the Run Type that is defined in O2 in "DataFormatsParameters/ECSDataAdapters.h". 
Concerning the beam type I am still waiting for the information.

Reply:
ok, found it, added doc.